includes/rest-api.php

```diff
@@ -720,8 +720,14 @@ public function validate_me( $request ) {
 		public function pmpro_rest_api_recent_memberships( $request ) {
 			$params = $request->get_params();
 
+			if ( isset($params['limit']) ) {
+				$members_limit = intval( $params['limit'] );
+			} else {
+				$members_limit = 1;
+			}
+
 			$response_type = isset( $params['response_type'] ) ? sanitize_text_field( $params['response_type'] ) : null;
-			$limit = apply_filters( 'pmpro_trigger_recent_members_limit', 1 );
+			$limit = apply_filters( 'pmpro_rest_api_recent_memberships_limit', $members_limit );
```

Comment:
Changing the filter name would be good long term but was there a specific reason to rename it here? If we rename it would at least need a backwards compatible filter call to the old name too.

It sounds like this might be better to leave as it was until we get down the road into refactoring more of our hooks at once.

```suggestion
			$limit = apply_filters( 'pmpro_trigger_recent_members_limit', $members_limit );
```